build.gradle

```diff
@@ -12,7 +12,7 @@ ext {
     coroutinesVersion = '1.5.2'
     androidxWorkVersion = "2.7.0"
 
-    fluxCVersion = 'trunk-f72e5d145dcfcf6e6875863e874496c1866676e7'
+    fluxCVersion = 'trunk-0b70d069111327513c6c54d05a362d971a555e0d'
```

Comment:
For refs for people coming back to that PR in the future: this is the revert we talked about in the comments above, see [#2432](https://github.com/wordpress-mobile/WordPress-FluxC-Android/issues/2432)